frontend/components/Header/Nav/Pillars/index.js

```diff
@@ -12,97 +12,98 @@ import { pillars as pillarColours } from '@guardian/pasteup/palette';
 import { headline } from '@guardian/pasteup/fonts';
 import type { LinkType } from '../__config__';
 
-const Pillars = styled('ul')({
-    clear: 'right',
-    margin: 0,
-    listStyle: 'none',
-    listStyleImage: 'none',
-    [mobileLandscape]: {
-        paddingLeft: 20,
-    },
+const pillarsStyles = css`
+    clear: right;
+    margin: 0;
+    list-style: none;
+    list-style-image: none;
+    ${mobileLandscape} {
```

Comment:
I don't think it's clear to use `${}` for media queries.

Reply:
Do you feel this is because the syntax is still unfamiliar? 

I think it's clear what the intention is here, although maybe someone new to the codebase might not be able to guess what the syntax for defining media queries might be?

Can you think of a syntax that might be clearer?

Reply:
Is it the tagged template literal `${}` syntax you feel is unclear or the variable name `mobileLandscape`? With documentation we could clarify how to do incorporate media queries to TTL style definitions. For reference the current `.scss` stylesheet use the `sass-mq` mixin, which would use the syntax ` @include mq(mobileLandscape)`.

Reply:
Yeah we could improve the naming to give a bit more indication it's a media query - perhaps an 'mq' prefix? But the general approach seems sound.

Reply:
I've added a trello card about this to the backlog. But I think the `from[x].until[y]` could be better served as `mq({from: x, until: y})`

I don't really have a problem with this though. I just thought I'd update this thread.

Reply:
I think if we can avoid it, we shouldn't provide helpers for ad-hoc breakpoints as it is hardly ever likely to be the 'right' thing.